pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "envier"
 dynamic = ["version"]
 description = "Python application configuration via the environment"
 readme = "README.md"
-license = "MIT"
+license = {file = "LICENSE"}
```

Comment:
Should we set the license in the classifiers field? I think that might be best practice: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers

Reply:
We can keep this change